misc-tools/configure-domjudge

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+
+'''
+TODO
+(replace main with the DOMjudge major.minor version if you are running a released version,
```

Comment:
What main?

Reply:
After I resolved the Todo the line above it will be clear.